app/api/ms_api_utils.py

```diff
@@ -22,18 +19,30 @@
 
 # create instance
 def post_request(request_url, data):
+    
     try:
 
         response = requests.post(
             request_url, json=data, headers={"Accept": "application/json"}
         )
         response.raise_for_status()
-    except requests.ConnectionError as e:
-        return f"{e.response.json()}", HTTPStatus.INTERNAL_SERVER_ERROR
-    except requests.HTTPError as e:
-        return f"{e.response.json()}", e.response.status_code
+        response_message = response.json()
+        response_code = response.status_code
+    except requests.exceptions.ConnectionError as e:
+        response_message = messages.INTERNAL_SERVER_ERROR
+        response_code = json.dumps(HTTPStatus.INTERNAL_SERVER_ERROR)
+        logging.warning(f"{e}")
+        return response_message, response_code
+    except requests.exceptions.HTTPError as e:
+        response_message = e.response.json()
+        response_code = e.response.status_code
+        return response_message, response_code
     except Exception as e:
```

Comment:
Don’t catch exception. You’re either expecting a specific failure, in that case catch it else you let it break and should fix it. https://realpython.com/the-most-diabolical-python-antipattern/

Reply:
mmm, I did this since trying to mock the exception is too difficult  as I can't predict what event to mock and there's no clear information to what type of response is returned and how to mock them in the test. 
I was thinking by doing this (returning a standard response to user and logging actual response to console): 
a> at least will mask the underlying issue on the UI but developers can further check this 500 response through the console and still will be able to create the ticket based on it. 
b> It is easier to mock the data that will be returned to User rather than trying to mock the random 500 event and its potential responses

Will this be ok with you or should I just skip testing Exception in this case? If we still need to find a way to test mock the exception without masking the response like I've done here, can you please open an issue for it for us to work on later pass MVP (as `a nice to have` test?)